torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -268,13 +268,21 @@ bool FunctionSignature::parse(PyObject* args, PyObject* kwargs, PyObject* dst[],
   auto nargs = PyTuple_GET_SIZE(args);
   ssize_t remaining_kwargs = kwargs ? PyDict_Size(kwargs) : 0;
   ssize_t arg_pos = 0;
+  bool allow_varargs_intlist = false;
 
   if (nargs > max_pos_args) {
-    if (raise_exception) {
-      // foo() takes takes 2 positional arguments but 3 were given
-      extra_args(*this, nargs);
+    // if there is a single positional IntList argument, i.e. expand(..), view(...),
```

Comment:
I don't see how this handles `view(1)`. In that case the containing if-statement `nargs > max_pos_args` won't be true because both `nargs` and `max_pos_args` will be 1.

Reply:
I checked the `torch.Size` case and it works because it ends up as an IntList anyway.  I'll add a case to `test_autograd`.